Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -1353,6 +1378,9 @@ static bool pathIsBlocked(const char* path)
     if (!g_path_is_absolute(path))
         return true;
 
+    if (pathIsUserHomeDirectory(path))
+        return true;
+
```

Comment:
This is unrelated to this patch, no? I would move it to its own bug and patch.

Reply:
OK.